client/src/components/dollarsignsandCategories.jsx

```diff
@@ -0,0 +1,43 @@
+import styled, { css } from 'styled-components';
+import React from 'react'
+
+const Dollars = styled.div`
+  line-height: 21px;
+`
+
+const DollarsDot = styled.i`
+  font-size: 0.25rem;
+  display: inline;
+  color: #666666;
+  top: -10px;
+  margin: 1px;
+  padding-left: 3px;
+  vertical-align: middle;
+`
+
+const DollarSignsAndCategories = (props) => {
+
+  let dollarSigns = ""
+  for (let i = 0; i < props.dollarSigns; i++){
+    dollarSigns += "$"
+  }
+
+  console.log("category props", props.categories);
```

Comment:
console.log!
i still have plenty i need to remove as well, just wanted to point it out to not let them slip though :)

Reply:
yeah i gotta get rid of these lol